src/include/basil.h

```diff
@@ -83,6 +83,9 @@
  * XML elements, attributes, and attribute values in BASIL.
  */
 
+#define CDATA_START "<![CDATA[\n"
+#define CDATA_END "]]>\n"
```

Comment:
The original intent of basil.h was for workload management packages to incorporate what Cray supplied. Given the Altair copyright above I must assume the changes did not come from Cray, in which case I have some suggestions.
- Everything in this file is specific to BASIL. We should not be defining macros for standard XML components (like CDATA) here.
- Name collisions are more likely if these macros are not prefixed by "BASIL_"
- A _BASIL_H macro should be added to prevent multiple inclusion.